API Google Analytics/HelloAnalytics.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+
+from apiclient.discovery import build
+from oauth2client.service_account import ServiceAccountCredentials
+
+import httplib2
+from oauth2client import client
+from oauth2client import file
+from oauth2client import tools
+
+SCOPES = ['https://www.googleapis.com/auth/analytics.readonly']
+DISCOVERY_URI = ('https://analyticsreporting.googleapis.com/$discovery/rest')
+KEY_FILE_LOCATION = 'client_secrets.p12'
+SERVICE_ACCOUNT_EMAIL = ''
+VIEW_ID = ''
+
```

Comment:
Why use list on line 14, tuple on line 15 and standart string on line 16? Besides, I'd prefer to include this information inside a authentication function. I'm not a big fan of global variables, makes it harder to understand the code